getpaid/backends/pagseguro/__init__.py

```diff
@@ -0,0 +1,204 @@
+
+    gateway_url_payment: url to redirect the client to pay the bill.
+    """
+
+    BACKEND = 'getpaid.backends.pagseguro'
+    BACKEND_NAME = _('PagSeguro')
+    BACKEND_ACCEPTED_CURRENCY = ('BRL', )
+    BACKEND_LOGO_URL = 'getpaid/backends/pagseguro/pag-seguro-logo.png'
+
+    _GATEWAY_URL_START_TRANSAC = "https://ws.pagseguro.uol.com.br/v2/checkout/?"
+    _GATEWAY_URL_PAYMENT = "https://pagseguro.uol.com.br/v2/checkout/payment.html?code="
+    _TRANSACTION_CONSULT_URL = "https://ws.pagseguro.uol.com.br/v2/transactions/notifications/"
+
+    _DEV_GATEWAY_URL_START_TRANSAC = "https://ws.sandbox.pagseguro.uol.com.br/v2/checkout/?"
+    _DEV_GATEWAY_URL_PAYMENT = "https://sandbox.pagseguro.uol.com.br/v2/checkout/payment.html?code="
+    _DEV_TRANSACTION_CONSULT_URL = "https://ws.sandbox.pagseguro.uol.com.br/v2/transactions/notifications/"
+
+    _USER_DATA_TO_PAGSEGURO = {
+        'name': 'Nome',  # Nome completo do cliente.
+        'address': 'Logradouro',  # Logradouro do cliente (ex: Rua, Av, etc.)
+        'address_number': 'Numero',  # Nº do imóvel do cliente (ex: 12)
+        'address_complement': 'Complemento',  # Complemento (ex: Sala 109 ou Casa 1)
+        'address_zip_code': 'CEP',  # O CEP de 8 dígitos do cliente. Somente números (ex: 22345678)
+        'address_quarter': 'Bairro',  # Bairro do cliente
+        'address_city': 'Cidade',  # Cidade do cliente (ex: São Paulo)
+        'address_state': 'Estado',  # Estado do cliente (SP) no formato duas letras
+        'phone': 'TelefoneFixo',  # Telefone fixo do cliente
+        'cliente_email': 'Email',  # E-mail do cliente
+    }
+
+    _ACOUNT_DATA = {
+        "email": "",
+        "token":""
+    }
+
+    _PRODUCT_DATA = {
+        "currency": "BRL",
+        "itemId1":"",
+        "itemDescription1":"",
+        "itemQuantity1":"1",
+        "itemAmount1":"",
+        "reference":"",
+    }
+
+
+    def get_gateway_url(self, request):
+        """  """
+        payp = PaymentProcessor
+        gateway_url = payp.get_backend_setting('gateway_url_start_transac',
+                                            self._GATEWAY_URL_START_TRANSAC)
+        gateway_url2 = payp.get_backend_setting('gateway_url_payment', 
+                                                self._GATEWAY_URL_PAYMENT)
+        
+        is_dev = payp.get_backend_setting('testing', False)
+        if is_dev:
+            gateway_url = payp.get_backend_setting('dev_gateway_url_start_transac',
+                                            self._DEV_GATEWAY_URL_START_TRANSAC)
+            gateway_url2 = payp.get_backend_setting('dev_gateway_url_payment', 
+                                                self._DEV_GATEWAY_URL_PAYMENT)
+
+        order = self.payment.order
+
+        token = PaymentProcessor.get_backend_setting('token')
+        email = PaymentProcessor.get_backend_setting('email')
+        self._ACOUNT_DATA["token"] = token
+        self._ACOUNT_DATA["email"] = email
```

Comment:
what's the point of using capital letters here? you're not using it as a const so it should stay as `self._account_data` or even `account_data` because you're using it in `get_gateway_url` method scope only